linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/combined_client_graph.js

```diff
@@ -4,11 +4,12 @@ var CombinedClientGraph = (function() {
   }
 
   return function(metricsCollector, routerName, $root, colors) {
+    var clientColors = colors;
     var query = Query.clientQuery().withRouter(routerName).withMetric("requests").build();
 
-    var timeseriesParams = function() {
+    function timeseriesParams(name) {
       return {
-        strokeStyle: colors[colorIdx++ % colors.length],
+        strokeStyle: clientColors[name.replace("requests", "")].color,
```

Comment:
I prefer that we don't try parsing metrics in multiple places. Consider something like:
strokeStyle: clientColors[name.match(Query.clientQuery().build())[1]].color